### PR TITLE
throwing errors for empty telemetry values

### PR DESCRIPTION
**Pull request by elopezanaya**
we have detected in telemetry several entries without values for OrgId, WidgetId and OrgUrl.

currently use optionals for those values, allowing empty telemtry values

in this PR I'm addressing that issue

- add validation for values of the required properties
- add missing values for mocks and tests, causing telemetry to aggregate empty values in kusto

### CODE REVIEW

File: chat-widget/stories/OmniChannelLiveChatWidget.stories.tsx [hunk 1]
```diff
@@ -20,6 +20,17 @@ const LiveChatWidgetTemplate: Story<ILiveChatWidgetProps> = (args) => <LiveChatW
 
 export const Default = LiveChatWidgetTemplate.bind({});
 
+const telemetryConfigGlobal = {
+
+    appId: "00000000-0000-0000-0000-000000000000",
+    orgId: "00000000-0000-0000-0000-000000000000",
+    orgUrl: "https://contoso.crm.dynamics.com",
+    telemetryDisabled:true,
```
Comment (xTEddie) [inline chat-widget/stories/OmniChannelLiveChatWidget.stories.tsx:28]: nit: spacing

File: chat-widget/src/components/livechatwidget/common/registerTelemetryLoggers.ts [hunk 2]
```diff
@@ -25,13 +26,27 @@ export const registerTelemetryLoggers = (props: ILiveChatWidgetProps, dispatch:
...
+        }
+
+        if (!props.chatSDK?.omnichannelConfig?.orgUrl || props.chatSDK?.omnichannelConfig?.orgUrl.trim().length === 0  ) {
+            throw new Error("orgUrl is undefined in ChatSDK");
+        }
+
         telemetryData.OCChatSDKVersion = telemetryConfig.OCChatSDKVersion ?? "0.0.0-0";
         telemetryData.chatComponentVersion = telemetryConfig.chatComponentVersion ?? "0.0.0-0";
         telemetryData.chatWidgetVersion = telemetryConfig.chatWidgetVersion ?? "0.0.0-0";
         telemetryData.orgId = props.chatSDK?.omnichannelConfig?.orgId;
```
Comment (xTEddie) [inline chat-widget/src/components/livechatwidget/common/registerTelemetryLoggers.ts:45]: It looks like `orgId`, `widgetId`, `orgUrl` is always retrieved from `ChatSDK`. Would it be possible to not expose these properties anymore?

Comment (elopezanaya) [reply chat-widget/src/components/livechatwidget/common/registerTelemetryLoggers.ts:45]: I'll create a task for it

**Review by charliewang95 — APPROVED** [review]

**Review by xTEddie — APPROVED** [review]
